docs/Documentation/UserGuide/9-Tools-Grafana.md

```diff
@@ -0,0 +1,139 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+<!-- TOC -->
+## Outline
+
+- IoTDB-Grafana
+    - Grafana installation
+        - Install Grafana
+        - Install data source plugin
+        - Start Grafana
+    - IoTDB installation
+    - IoTDB-Grafana installation
+        - Start IoTDB-Grafana
+    - Explore in Grafana
+        - Add data source
+        - Design in dashboard
+
+<!-- /TOC -->
+# IoTDB-Grafana
+
+This project provides a connector which reads data from IoTDB and sends to Grafana(https://grafana.com/). Before you use this tool, make sure Grafana and IoTDB are correctly installed and started.
+
+## Grafana installation
+
+### Install Grafana
+
+* Download url: https://grafana.com/grafana/download
+* version >= 4.4.1
+
+### Install data source plugin
+
+* plugin name: simple-json-datasource
+* Download url: https://github.com/grafana/simple-json-datasource
+
+After downloading this plugin, you can use the grafana-cli tool to install SimpleJson from the commandline:
+
+```
+grafana-cli plugins install grafana-simple-json-datasource
+```
+
+Alternatively, you can manually download the .zip file and unpack it into your grafana plugins directory.
+
+* `{grafana-install-directory}\data\plugin\` (Windows)
+* `/var/lib/grafana/plugins` (Linux)
+* `/usr/local/var/lib/grafana/plugins`(Mac)
+
+### Start Grafana
+If you use Unix, Grafana will auto start after installing, or you can run `sudo service grafana-server start` command. See more information [here](http://docs.grafana.org/installation/debian/).
+
+If you use Mac and `homebrew` to install Grafana, you can use `homebrew` to start Grafana.
+First make sure homebrew/services is installed by running `brew tap homebrew/services`, then start Grafana using: `brew services start grafana`.
+See more information [here](http://docs.grafana.org/installation/mac/).
+
+If you use Windows, start Grafana by executing grafana-server.exe, located in the bin directory, preferably from the command line. See more information [here](http://docs.grafana.org/installation/windows/).
+
+## IoTDB installation
+
+See https://github.com/apache/incubator-iotdb
```

Comment:
How about changing this to an internal link to the corresponding chapter in the user guide.